// Test prototype functions
log("Prototype functions:")
log()

let list = [100, 200, 300, (x) => {return x}]
log("list: ", list)
log()

let sevens = 777
log("let sevens = 777; list->append(sevens, 888);")
list->append(sevens, 888)
log("list: ", list)
log("listLen = list->length()")
let listLen = list->length()
log("listLen: ", listLen)


log()
log()


let obj = {
	num: 1234,
	nested: {
		num2: 4444
	}
}
log("obj: ", obj)
log()

log('obj.nested->remove("num2")')
obj.nested->remove("num2")
log("obj: ", obj)
log()

log('obj->tryRemove("itsNotHere")')
obj->tryRemove("itsNotHere") // No error, since it's just a try.
log("obj: ", obj)
log()

log('obj->tryRemove("num", "nested")')
obj->tryRemove("num", "nested") // Remove both.
log("obj: ", obj)
